pkgs/top-level/go-packages.nix

```diff
@@ -2898,24 +2890,23 @@ let
       yaml-v2
     ];
 
+    doCheck = true;
+
+    buildFlagsArray = ''
+      -ldflags=
+          -X main.buildVersion=${rev}
```

Comment:
These don't match what prometheus uses upstream - does it still show the right metadata on the status page at runtime?

(https://github.com/prometheus/prometheus/blob/0.17.0rc2/scripts/build.sh#L32)